challenge_13/python/sarcodian/src/challenge_13.py

```diff
@@ -0,0 +1,30 @@
+def int_palin():
+    '''
```

Comment:
You could just reverse the number without getting the length and check if that's equal to the original number.